regolith/helpers/l_membershelper.py

```diff
@@ -67,15 +67,28 @@ def sout(self):
         bad_stati = ["finished", "cancelled", "paused", "back_burner"]
         people = []
         for person in self.gtx["people"]:
-            if rc.current and not person.get('active'):
-                continue
-            people.append(person)
-        #people.sort()
+            if rc.current:
+                if not person.get('active'):
+                    continue
+                people.append(person)
+            elif rc.former:
+                if person.get('active'):
+                    continue
+                people.append(person)
+            else:
+                people.append(person)
+
         for i in people:
             if rc.verbose:
                 print("{}, {} | group_id: {}".format(i.get('name'), i.get('position'), i.get('_id')))
                 print("    orcid: {} | github_id: {}".format(i.get('orcid_id'), i.get('github_id')))
+                for employment in i.get('employment'):
+                    if is_current(employment):
+                        print("    current organization: {}".format(employment.get('organization')))
+                        print("    current position: {}".format(employment.get('position')))
```

Comment:
This will print more nicely this way:
```
print("    current position: {}".format(position.get('full_position', position.get('position').title())))
```